.github/workflows/zip.yml

```diff
@@ -0,0 +1,162 @@
+name: 'Package Zip'
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      final:
+        description: 'Zip type: 0 = dev, 1 = production'
+        required: false
+        default: '0'
+jobs:
+  zip:
+    runs-on: ubuntu-latest
+    steps:
+      - name: If a pull_request, set final to 0
+        if: github.event_name == 'pull_request'
+        run: |
+          echo "JOB_FINAL=0" >> $GITHUB_ENV
+          echo "JOB_BRANCH=${{ github.event.pull_request.head.ref }}" >> $GITHUB_ENV
+      - name: If a workflow_dispatch, set final based on input
+        if: github.event_name == 'workflow_dispatch'
+        run: |
+          echo "JOB_FINAL=${{ github.event.inputs.final }}" >> $GITHUB_ENV
+          branch=$(echo "${{ github.ref }}" | sed -r "s|refs/heads/||" | sed -r "s|refs/tags/||")
+          echo "JOB_BRANCH=${branch}" >> $GITHUB_ENV
+      - name: Setup branch for consumption in subsequent steps
+        id: settings
+        run: |
+          echo "::set-output name=branch::${JOB_BRANCH}"
+      # ------------------------------------------------------------------------------
+      # Setup SSH keys and known_hosts
+      # ------------------------------------------------------------------------------
+      - name: Setup SSH keys and known_hosts
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+        run: |
+          mkdir -p ~/.ssh
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${{ secrets.GH_ACTIONS_SSH_KEY }}"
+      # ------------------------------------------------------------------------------
+      # Checkout the repo and tric
+      # ------------------------------------------------------------------------------
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1000
+          token: ${{ secrets.GH_BOT_TOKEN }}
+          submodules: recursive
+          ref: ${{ steps.settings.outputs.branch }}
+      - name: Checkout tut
+        uses: actions/checkout@v2
+        with:
+          repository: the-events-calendar/tut
+          ref: feature/use-node-modules-gulp
```

Comment:
Is it time to merge that PR and just reference `main` here?

Reply:
yup, i think so!